.gitignore

```diff
@@ -73,3 +73,4 @@ sonar-project.properties
 
 #Misc
 .DS_Store
+/.run/
```

Comment:
Sneaking this in cause I couldn't be bothered to raise a PR on it's. LMK if you care and I'll split it! :)

Reply:
I wont lose any sleep over this! 🤣